pkgs/by-name/re/readest/package.nix

```diff
@@ -0,0 +1,96 @@
+{
+  rustPlatform,
+  pnpm,
```

Comment:
Most likely you need to change `pnpm` to `pnpm_9`. Since #371832, it points to `pnpm_10` instead of `pnpm_9`.